### PR TITLE
fix(lnurlwithdraw): display error on prompt #1451

### PR DESCRIPTION
**Pull request by escapedcat**
#### Other LNURLActions

- LNURLAuth: aligning error-handling with other LNURLActions
- LNURLChannel: is already taking care of this
- LNURLPay: is already taking care of this

### Link this PR to an issue

#1451

### Type of change (Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Tested with LNMarkets (see screenshot)

### CODE REVIEW

File: src/app/screens/LNURLWithdraw/index.tsx [hunk 1]
```diff
@@ -62,15 +62,14 @@ function LNURLWithdraw() {
 
       if (response.data.status.toUpperCase() === "OK") {
         setSuccessMessage("Withdraw request sent successfully.");
+        // ATTENTION: if this LNURL is called through `webln.lnurl` then we immediately return and return the response. This closes the window which means the user will NOT see the above successAction.
```
Comment (escapedcat) [inline src/app/screens/LNURLWithdraw/index.tsx:65]: @bumi this seems to do the trick. Do you see any issues with this?  
I could apply this to the other prompts.

Comment (bumi) [reply src/app/screens/LNURLWithdraw/index.tsx:65]: yep, I guess something like that 👍 
either moving the reply up. or extending the `if (navState.isPrompt) {` around the reply.

**Review by bumi — APPROVED** [review]

File: src/app/screens/LNURLAuth/index.tsx [hunk 2]
```diff
@@ -50,19 +51,18 @@ function LNURLAuth() {
...
-      // We assume this is OK when it is called through webln.
-      if (navState.isPrompt) {
-        msg.reply(response);
+        toast.error(`Error: Auth status is not ok`);
       }
     } catch (e) {
       console.error(e);
       if (e instanceof Error) {
-        toast.error(`Error: ${e.message}`);
+        setErrorMessage(`Error: ${e.message}`);
```
Comment (bumi) [inline src/app/screens/LNURLAuth/index.tsx:65]: above we're using toast and here we set an error message. should those be in sync?

**Review by bumi — APPROVED** [review]

File: src/app/screens/LNURLAuth/index.tsx (hunk 2 above)
Comment (escapedcat) [reply src/app/screens/LNURLAuth/index.tsx:64]: Yes, changed it. I think currently we use the toast if i.e. a call fails. For other errors we can use the error-message.

**Pull request merged**